Libigl incompatible with recent imgui

Libigl does not compile with newer versions of imgui:

    ../third-parties/libigl/libigl/include/igl/opengl/glfw/imgui/ImGuiHelpers.h:61:91: error: unknown type name 'ImGuiTextEditCallback'; did you mean 'ImGuiInputTextCallback'?
    inline bool InputText(const char* label, std::string &str, ImGuiInputTextFlags flags = 0, ImGuiTextEditCallback callback = NULL, void* user_data = NULL)
                                                                                              ^~~~~~~~~~~~~~~~~~~~~
                                                                                              ImGuiInputTextCallback
    ../third-parties/imgui/imgui/../imgui/imgui.h:199:15: note: 'ImGuiInputTextCallback' declared here
    typedef int (*ImGuiInputTextCallback)(ImGuiInputTextCallbackData* data);    // Callback function for ImGui::InputText()

Indeed, `ImGuiTextEditCallback` was renamed to `ImGuiInputTextCallback`...

#### Applies to:
- [x] Windows
- [x] macOS
- [x] Linux

--- a/include/igl/opengl/glfw/imgui/ImGuiHelpers.h
+++ b/include/igl/opengl/glfw/imgui/ImGuiHelpers.h
@@ -58,7 +58,7 @@ inline bool ListBox(const char* label, int* idx, std::vector<std::string>& value
     static_cast<void*>(&values), values.size());
 }
 
-inline bool InputText(const char* label, std::string &str, ImGuiInputTextFlags flags = 0, ImGuiTextEditCallback callback = NULL, void* user_data = NULL)
+inline bool InputText(const char* label, std::string &str, ImGuiInputTextFlags flags = 0, ImGuiInputTextCallback callback = NULL, void* user_data = NULL)
 {
   char buf[1024];
   std::fill_n(buf, 1024, 0);

